.github/workflows/docker-image.yml

```
@@ -0,0 +1,31 @@
+name: deploy-docker
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+
+jobs:
+  build:
+
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    
+    steps:
+      - name: Checkout repository
+        uses: appleboy/ssh-action@v0.1.4
+        with:
+          host: "${{ secrets.DO_DEPLOY_IP }}"
+          username: "${{ secrets.DO_DEPLOY_USERNAME }}"
+          key: "${{ secrets.DO_DEPLOY_KEY }}"
```

```suggestion
          host: ${{ secrets.DO_DEPLOY_IP }}
          username: ${{ secrets.DO_DEPLOY_USERNAME }}
          key: ${{ secrets.DO_DEPLOY_KEY }}
```
I think these shouldn't have the quotes -- might be why the CI is failing currently?